server/keyshare/keyshareserver/server.go

```diff
@@ -279,17 +278,13 @@ func (s *Server) generateResponse(ctx context.Context, user *User, authorization
 		return "", errMissingCommitment
 	}
 
-	// Indicate activity on user account
-	err := s.db.setSeen(ctx, user)
-	if err != nil {
-		s.conf.Logger.WithField("error", err).Error("Could not mark user as seen recently")
-		// Do not send to user
-	}
+	// Indicate activity on user account. Do not send error to user.
+	_ = s.db.setSeen(ctx, user)
```

Comment:
why not just also remove `_ = `

Reply:
I'm a favour of explicitly showing that I drop a variable here. That makes it more clear in the future that we make an assumption here. If you have to add code in the future, then that might be important.

But indeed, it's just one of my habits. We would't get a warning if we remove `_ =`.